extensions/amp-story/amp-story-bookend.md

```diff
@@ -0,0 +1,304 @@
+---
+$category@: presentation
+formats:
+  - websites
```

Comment:
I think websites should be removed - unless bookends have a functionality that is website specific?